core/src/main/java/org/apache/accumulo/core/file/rfile/RFileOperations.java

```diff
@@ -131,6 +135,20 @@ protected FileSKVWriter openWriter(FileOptions options) throws IOException {
       outputStream = fs.create(new Path(file), false, bufferSize, (short) rep, block);
     }
 
+    if (options.dropCacheBehind) {
+      // Tell the DataNode that the write ahead log does not need to be cached in the OS page
+      // cache
```

Comment:
copy / paste error? This isn't a write-ahead log.

Reply:
Yes, good catch. Will fix.

Reply:
Resolved in 4cd1525